pick.c

```diff
@@ -456,6 +458,16 @@ selected_choice(void)
 			else
 				yscroll = selection = 0;
 			break;
+		case HOME:
+			if (selection > 0)
+				yscroll = selection = 0;
+			break;
+		case END:
+			if (selection < choices_count - 1)
+				selection = choices_count - 1;
+				if (selection - yscroll >= choices_lines)
```

Comment:
See comment above, this line also has misleading indentation since the if-statement above is not wrapped in curly-braces.